src/modules/ssml-timeline.ts

```diff
@@ -336,14 +336,20 @@ export class SsmlTimeline extends PolymerElement {
     tracks.style.width = `${max}px`;
 
 
-    for (let i = 0; i < max; i += 40) {
-      // Put something every 40 pixels
-      tickerHtml += '<div part=\'ticker-tick\' class=\'ticker-tick\'>'
-      if (i % this.pixelsPerSecond === 0) {
-        // Perfect second
-        tickerHtml += `${i / this.pixelsPerSecond}s`;
-      } else {
-        tickerHtml += `|`
+    for (let i = 0; i < max; i += 50) {
+      // Put something every 50 pixels
+      tickerHtml += '<div part=\'ticker-tick\' class=\'ticker-tick\'>';
+      if(i == 0){
```

Comment:
Can you revise the formatting here, using 3 equal signs for equality and spaces between `if`, `else`, and brackets?

Reply:
@Fleker ok done